go/proxyd/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo "Updating CA certificates."
+update-ca-certificates
```

Comment:
Shouldn't this step happen at build time to make it reproducible?

Reply:
`update-ca-certificates` needs to be run whenever the supplied certificates change. Since we support bind-mounting the certificates at runtime, this needs to happen before executing the proxyd binary.

Reply:
Updating certs from the internet at runtime is asking for trouble.

Can we default to the container CA certs, but allow specifying a CA file at runtime (end drop the entrypoint script)?

Reply:
`update-ca-certificates` doesn't update certs from the internet, it just appends all certs in the `ca-certificates` directory into one big file in `/etc/ssl`: https://gitlab-test.alpinelinux.org/alpine/ca-certificates/-/blob/master/update-ca.c.

That said, I think it's also useful to specify a CA file at runtime for things like client certificates, so I've added that functionality.

Reply:
Got it, I thought that `update-ca-certificates` called out to the network and downloaded the latest certs from the alpine registry